src/protocols/displayport.stanza

```diff
@@ -54,16 +57,38 @@ doc: \<DOC>
 @brief DisplayPort Bundle
 The Data Bundle consists of 4 lane pairs (1 RX or TX pair set = 1 lane)
 The bundle also constrains an AUX channel (differential pair) and a HPD pin
+
+@member aux Differential pair for the auxiliary channel
+@member lane Four differential pairs for data transfer
+@member hpd Hot plug detect
 <DOC>
 
 public pcb-bundle displayport :
   name = "DisplayPort"
   description = "DisplayPort Digital Display Interface"
   port aux  : diff-pair
-  port lane : diff-pair[4]
+  port lane : diff-pair[DISPLAYPORT_NUM_LANES]
```

Comment:
We discussed that there is a miniature display port with fewer lanes - correct ? 

Should this not be an argument with default value ?

Reply:
I don't recall that we did. If there is an option for fewer number of lanes, I can make this an argument. AFAIK, miniDP is still 4 lanes.

Reply:
No - then nevermind.